dm/dm-ansible/conf/task.yaml.example

```diff
@@ -69,7 +69,7 @@ mysql-instances:             # one or more source database, config more source d
 
     loader:                  # local loader rule
       pool-size: 16
-      dir: "./dumped_data"   # must be unique between tasks for the same instance
+      dir: "./dumped_data"   # if not ends with `.name`, then `.name` will be appended
```

Comment:
explain `name` is the name variable in task.yaml ?

Reply:
explained in https://github.com/pingcap/dm/pull/183/commits/b7aba30a615bb13a078766f9a8343b3730d31d15.